docs/en/latest/how-to-build.md

```diff
@@ -167,3 +167,15 @@ Content-Type: text/html
 Some features require you to build OpenResty with extra Nginx modules.
 If you need those features, you can build OpenResty with
 [this build script](https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh).
+
+## 7. Add systemd unit file for APISIX
+
+If you install APISIX with rpm package, the unit file is installed automatically, and you could directly do
+
+```
+$ systemctl start apisix
+$ systemctl stop apisix
+$ systemctl enable apisix
+```
+
+If installed in other methods, you could refer to [the unit file template](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service), modify if needed, and place it as `usr/lib/systemd/system/apisix.service`.
```

Comment:
Bad path

Reply:
The path would work when PR in api7 merged.

Reply:
I think you should use path in the file system, not the relative one in the build tool repo.

Reply:
Installed in other methods, maybe don't have `apisix.service` in the file system.

Reply:
> I think you should use path in the file system, not the relative one in the build tool repo.

Got it. Also add a note to indicate another copy exists, so when people change it, they would be aware of the need of changing the extra copy.

Reply:
@Yiyiyimu 
Actually, I mean the service file should be put into the deployed machine's `/usr/lib/systemd/system/apisix.service`, but not in this repo.

Reply:
> Actually, I mean the service file should be put into the deployed machine's /usr/lib/systemd/system/apisix.service, but not in this repo.

lol got it 🤣 fixed